### DISCUSSION TITLE
Paste Image in Source editor

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#7623] [Feature Requests]
### Description

I quite love that I can paste an image from my clipboard into the visual editor and it saves the image into `./images`. But I cannot do this using the source editor. I would reallyyyy like to! Mostly because I do not like how the visual editor edits my file and adds additional spaces between lines.

**Comment by mcanouil** [comment]
"source editor" of which IDE? RStudio? VSCode? Neovim? something else?

Repository to target depending on the answer:
- Visual Editor (independently of the IDE): https://github.com/quarto-dev/quarto
- Visual Studio Code Quarto extension: https://github.com/quarto-dev/quarto
- RStudio: https://github.com/rstudio/rstudio
